elasticapm/transport/base.py

```diff
@@ -62,6 +62,7 @@ class Transport(object):
 
     def __init__(
         self,
+        client,
```

Comment:
Any particular reason this is a required arg rather than `client=None`?

Reply:
I made it a positional argument to indicate that it is required for proper functionality. It adds a bit of code churn in some tests where we now have to explicitly pass `None` as client, but I'm personally fine with that. But no strong opinions, I can change it if you want

Reply:
Sounds fine.